src/teleport.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 #ifndef CATA_SRC_TELEPORT_H
 #define CATA_SRC_TELEPORT_H
+#include "point.h"
```

Comment:
Forward declaration is sufficient for declaring `teleport_to_point`.
```suggestion
struct tripoint;
```